recipes/libpointmatcher/all/conanfile.py

```diff
@@ -41,15 +51,19 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+        if is_msvc(self):
+            # Only static for msvc
+            del self.options.shared
+            self.package_type = "static-library"
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
         self.requires("boost/1.81.0", transitive_headers=True)
-        # FIXME: eigen 3.4.0 is not compatible yet (see https://github.com/ethz-asl/libpointmatcher/issues/496)
-        self.requires("eigen/3.3.9", transitive_headers=True)
+        self.requires("eigen/3.4.0", transitive_headers=True)
         self.requires("libnabo/1.0.7")
+        self.requires("yaml-cpp/0.7.0", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("yaml-cpp/0.8.0", transitive_headers=True)
```
Unless the v0.7.0 is to avoid a version conflict, v0.8.0 is newer and more common on CCI.

Reply:
It is the required version upstream. 
I'll check on Monday if 0.8.0 works. 
If it do, we can bump!
Thanks 😃